src/middleware/graphql.ts

```diff
@@ -197,36 +295,39 @@ export abstract class GraphQLBase implements Initializable<express.Application>
         }),
       formatError: (error: GraphQLError) => {
         let formattedError = error;
-
         try {
-          // Inherits any `isExpected` flag present on the `originalError`
           formattedError = GraphQLErrorFormatter.format(error);
         } catch (err) {
           log.error(`An unexpected error occurred while formatting the GraphQL error`, err);
         }
         const errorMessage = String(formattedError);
-        const { originalError } = formattedError; // Can be undefined
-
-        const extra = {
-          'GraphQL Body': _.get(formattedError, 'source.body', undefined)
-        };
-
-        if (originalError && originalError instanceof NestedError) {
-          Object.assign(extra, (originalError as NestedError).getSentryExtraData())
+        const { originalError } = error;
+
+        // Kind of odd, but originalError can be undefined
+        if (_.get(originalError, 'isExpected', false)) {
+          log.info(`Trapped expected error: ${errorMessage}\n${formattedError}`);
+        } else {
+          log.error(errorMessage, formattedError);
+
+          if (ErrorReporter.isInitialized()) {
+            ErrorReporter.capture(errorMessage, {
+              exception: formattedError,
+              extra: {
+                path: formattedError.path,
+                positions: formattedError.positions,
+                source: formattedError.source && formattedError.source.body
+              }
+            });
+          }
         }
 
-        // Implicitly logs the error to the console
-        ErrorReporter.capture(errorMessage, {
-          exception: formattedError,
-          extra
-        });
-
         return formattedError;
       },
       introspection: true,
       playground: {
         endpoint: `${this.path}`,
         settings: {
+          'editor.cursorShape': 'line',
```

Comment:
Is this related to the core purpose of this PR, or should it be done separately?